packages/react/src/components/switch/switch.stories.tsx

```diff
@@ -24,51 +24,50 @@ export const Basic: Story = () => {
   return <Switch>basic</Switch>
 }
 
-export const WithReverse: Story = () => {
+export const Reverse: Story = () => {
   return <Switch reverse>basic</Switch>
 }
 
-export const WithSize: Story = () => {
+export const Size: Story = () => {
   return (
-    <>
-      <Switch size="sm">small size</Switch>
-      <Switch size="md">medium size</Switch>
-      <Switch size="lg">large size</Switch>
-    </>
+    <PropsTable variant="column" rows={["sm", "md", "lg"]}>
+      {(_, row, key) => <Switch key={key} size={row}>{`${row} size`}</Switch>}
+    </PropsTable>
   )
 }
 
-export const WithVariant: Story = () => {
+export const Variant: Story = () => {
   return (
-    <>
-      <Switch variant="thick">thick</Switch>
-      <Switch variant="thin">thin</Switch>
-    </>
+    <PropsTable variant="column" rows={["thick", "thin"]}>
+      {(_, row, key) => (
+        <Switch key={key} variant={row}>
+          {row}
+        </Switch>
+      )}
+    </PropsTable>
   )
 }
 
-export const WithLabel: Story = () => {
+export const Label: Story = () => {
   const id = useId()
 
   return (
     <HStack gap="sm">
-      <Label htmlFor={id} userSelect="none">
+      <FieldLabel htmlFor={id} userSelect="none">
```

Comment:
If you do not use `<Field.Label>` here, you will get an error.

<img width="812" alt="image" src="https://github.com/user-attachments/assets/1a2b37ab-48d8-4d38-9461-41d74b7d2432" />

```suggestion
      <Field.Root>
        <Field.Label htmlFor={id} userSelect="none">
          Please Click
        </Field.Label>
      </Field.Root>
```